Dockerfile

```diff
@@ -23,6 +31,9 @@ COPY deepcell /opt/deepcell-tf/deepcell
 # Install deepcell via setup.py
 RUN pip install /opt/deepcell-tf
 
+## need to remove jedi - causes erros for tab in jupyter
+RUN pip uninstall jedi --yes
+
```

Comment:
What installs `jedi`? If possible it would be good to prevent this from getting installed since more Docker layers means larger Docker images.